packages/react-ui-core/src/Menu/Menu.js

```diff
@@ -173,7 +164,7 @@ export default class Menu extends PureComponent {
           className,
           theme.Menu
         )}
-        items={this.items}
+        items={options}
```

Comment:
are you sure this shouldn't be `this.options` ?

Reply:
Good catch! Yes it should be 😄